dfp/math.h

```diff
@@ -265,6 +265,15 @@ extern _Decimal32 __rintd32 (_Decimal32 __x) __THROW;
 extern _Decimal32 nextafterd32 (_Decimal32 __x, _Decimal32 __y) __THROW __attribute__ ((__const__));
 extern _Decimal32 __nextafterd32 (_Decimal32 __x, _Decimal32 __y) __THROW __attribute__ ((__const__));
 
+extern _Decimal32 nextupd32 (_Decimal32 __x) __THROW __attribute__ ((__const__));
+extern _Decimal32 __nextupd32 (_Decimal32 __x) __THROW __attribute__ ((__const__));
+
+extern _Decimal64 nextupd64 (_Decimal64 __x) __THROW __attribute__ ((__const__));
+extern _Decimal64 __nextupd64 (_Decimal64 __x) __THROW __attribute__ ((__const__));
+
+extern _Decimal128 nextupd128 (_Decimal128 __x) __THROW __attribute__ ((__const__));
+extern _Decimal128 __nextupd128 (_Decimal128 __x) __THROW __attribute__ ((__const__));
```

Comment:
The TS 18661-2 prototypes should be guarded by the definition of `__STDC_WANT_IEC_60559_DFP_EXT__`.

Reply:
Please give a look on this commit: https://github.com/libdfp/libdfp/pull/95/commits/88f0f10cf9f73578b901cdf52ce0f03a5066bb84

I've changed the Makefile to define this constant at compilation time. I think I'm cheating but I don't know how to do it in another way.